step-release-vis/src/app/components/environments/environments.html

```diff
@@ -24,101 +24,105 @@
       Reset
     </button>
   </div>
-  <div [ngStyle]="{display: 'inline-block', height: getTimelineHeight() + 'px'}">
-    <svg xmlns="http://www.w3.org/2000/svg"
-         id="top-timeline"
-         [attr.width]="envWidth"
-         [attr.height]="getTimelineHeight()"
-         [ngStyle]="{left: TITLE_WIDTH}">
-      <rect [attr.x]="0"
-            [attr.y]="getTimelineHeight() / 2 + 8"
-            [attr.width]="envWidth"
-            [attr.height]="2"
-            fill="gray"/>
-      <circle *ngIf="shouldDisplayTimelineCircle()"
-              id="cur-snapshot-line"
-              [attr.cx]="getLineX() - 1"
-              [attr.cy]="getTimelineHeight() / 2 + 9"
-              [attr.r]="3"
+  <div id="environments"
```

Comment:
why do we need the extra div? can't we just do this on the same area we handle the clicks already for the tooltip?

Reply:
Tooltip clicks are handled by child components, since the tooltip is a part of child. The time range choosing is global, so the listener is in parent comp.